library/pathname/empty_spec.rb

```diff
@@ -0,0 +1,28 @@
+require File.expand_path('../../spec_helper', __dir__)
+require 'pathname'
+
+describe "Pathname#empty?" do
+  ruby_version_is "2.4" do
+    it 'returns true when file is not empty' do
+      (Pathname.new(__FILE__).empty?).should be_false
+    end
+
+    it 'returns false when the directory is not empty' do
+      (Pathname.new(__dir__).empty?).should be_false
```

Comment:
Here I'm using the fact `__dir__`  is the directory of this same file (and should not be empty), if you need to explicitly create a non empty directory (as in `/core/dir/fixtures/common.rb`), I can change it.

Reply:
That's OK.